python_ta/reporters/plain_reporter.py

```diff
@@ -106,6 +107,10 @@ def print_messages(self, level='all'):
         print('\n')
 
     def sort_messages(self):
+        max_messages = float('inf')
```

Comment:
We can use the default set in `init_linter()` in `python_ta/__init__.py`
I didn't see an easy way to allow a setting for infinite messages, should I make a new issue for that?